exonum-java-binding-core/rust/src/runtime/config.rs

```diff
@@ -34,3 +40,74 @@ pub struct ServiceConfig {
     /// A port of the HTTP server for Java services. Must be distinct from the ports used by Exonum.
     pub port: i32,
 }
+
+/// Error returned while validating user-specified additional parameters for JVM.
+/// Trying to specify a parameter that is set by EJB internally.
+#[derive(PartialEq)]
+pub struct ForbiddenParameterError(String);
+
+impl fmt::Debug for ForbiddenParameterError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
+        write!(
+            f,
+            "Trying to specify JVM parameter [{}] that is set by EJB internally. \
+             Use EJB parameters instead.",
+            self.0
+        )
+    }
+}
+
+/// Checks if parameter is not in list of forbidden parameters and adds a dash at the beginning.
+pub(crate) fn validate_and_convert(
+    user_parameter: &str,
+) -> Result<String, ForbiddenParameterError> {
+    check_not_forbidden(user_parameter)?;
+    // adding dash at the beginning
+    let mut user_parameter = user_parameter.to_owned();
+    user_parameter.insert(0, '-');
+
+    Ok(user_parameter)
+}
+
+fn check_not_forbidden(user_parameter: &str) -> Result<(), ForbiddenParameterError> {
+    if user_parameter.contains("Djava.class.path")
```

Comment:
Isn't contains to restrictive (what if I want to specify a custom property, e.g., `-Dfoo.bar=bazDjava.class.path`)? Maybe starts_with?